modules/va_facilities/README.v1.yml

```diff
@@ -434,15 +434,14 @@ paths:
     get:
       tags:
         - facilities
-      summary: Query facilities based on drive time from a specified location
+      summary: Query health facilities based on drive time from a specified location
```

Comment:
Does `VHA` versus `health` make any semantic difference here?

Reply:
They are synonymous in my reading. I used "health" because that is what they were called in the sentence I modified in the next paragraph, but if VHA is more explicit I don't mind changing it.